OpenRA.Mods.Common/Widgets/CheckboxWidget.cs

```diff
@@ -54,17 +56,21 @@ public override void Draw()
 			var disabled = IsDisabled();
 			var font = Game.Renderer.Fonts[Font];
 			var color = GetColor();
+			var hover = Ui.MouseOverWidget == this;
 			var colordisabled = GetColorDisabled();
 			var bgDark = GetContrastColorDark();
 			var bgLight = GetContrastColorLight();
 			var rect = RenderBounds;
 			var text = GetText();
 			var textSize = font.Measure(text);
 			var check = new Rectangle(rect.Location, new Size(Bounds.Height, Bounds.Height));
-			var baseName = IsHighlighted() ? "checkbox-highlighted" : "checkbox";
-			var state = WidgetUtils.GetStatefulImageName(baseName, disabled, Depressed && HasPressedState, Ui.MouseOverWidget == this);
 
-			WidgetUtils.DrawPanel(state, check);
+			if (DrawPanel)
```

Comment:
:thinking: How about renaming that to `HasBackground`? `DrawPanel` is not very clear unless you know the method name.

Reply:
Sure